simple_calculator/index.html

```diff
@@ -11,27 +11,29 @@
     </head>
 
     <body>
-        <h1>New Calculator For Project!</h1>
-        <br/>
-        <form id="myform" Name="calc">
-        <input name="display" style="width:675px; height:100px; 
-        text-align:center; background-color:#1f80c9;"> <br/>
-        <input type=button value="0" Onclick="calc.display.value+='0'">
-        <input type=button value="1" Onclick="calc.display.value+='1'">
-        <input type=button value="2" Onclick="calc.display.value+='2'">
-        <input type=button value="+" Onclick="calc.display.value+='+'" style="background-color:#cc5c11"> <br/>
-        <input type=button value="3" Onclick="calc.display.value+='3'">
-        <input type=button value="4" Onclick="calc.display.value+='4'">
-        <input type=button value="5" Onclick="calc.display.value+='5'">
-        <input type=button value="-" Onclick="calc.display.value+='-'" style="background-color:#ba55d3"> <br/>
-        <input type=button value="6" Onclick="calc.display.value+='6'">
-        <input type=button value="7" Onclick="calc.display.value+='7'">
-        <input type=button value="8" Onclick="calc.display.value+='8'">
-        <input type=button value="x" Onclick="calc.display.value+='*'" style="background-color:#7db1b2;"> <br/>
-        <input type=button value="9" Onclick="calc.display.value+='9'">
-        <input type=button value="C" Onclick="calc.display.value=''" style="background-color:red;">
-        <input type=button value="=" Onclick="calc.display.value=eval(calc.display.value)" style="background-color:pink;">
-        <input type=button value="&#247;" Onclick="calc.display.value+='/'" style="background-color:green;">
+        <h1 style="text-align:center">New Calculator For Project!</h1>
+        <div class = 'cal-group'>
```

Comment:
1. 보통 띄어쓰기를 안넣는 게 일반적인 것 같습니다.
2. 그리고 다른 곳에선 쌍따옴표를 썼었으니 쌍따옴표로 통일하는 게 낫겠습니다.
```suggestion
        <div class="cal-group">
```

Reply:
새로운 commit으로 반영 완료